lecture_notes/08 collections/1-array.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
 const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];
-
+ages.splice(4, 1);
 const first = ages[0];
-const last = ages[ages.length - 1];
-
+const last = ages[2];
+ages.splice(4, 1);
+ages.splice(5, 1, 'ABOBA');
+ages.push('BIBA');
+ages.shift();
+ages.unshift('Start');
```

Comment:
Adding strings to array of numbers will work but it is not a good idea because each reading from array will require additional type checking